class/deeplink-schema-match.ts

```
@@ -34,172 +34,192 @@ class DeeplinkSchemaMatch {
    */
-  static navigationRouteFor(
+  static async navigationRouteFor(
     event: { url: string },
-    completionHandler: (args: TCompletionHandlerParams) => void,
     context: TContext = { wallets: [], saveToDisk: () => {}, addWallet: () => {}, setSharedCosigner: () => {} },
-  ) {
-    if (event.url === null) {
-      return;
-    }
-    if (typeof event.url !== 'string') {
+  ): Promise<TCompletionHandlerParams | void> {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)